docs/docs/middlewares/asset.md

```diff
@@ -0,0 +1,22 @@
+# Asset
+This component returns path to file. 
```

Comment:
I dont think it is component, but rather a function as per twig docs:
> Twig comes with a long list of tags, filters and functions that are available by default. You can even add your own custom filters, functions (and more) via a Twig Extension.

https://symfony.com/doc/current/templating.html#templates